### ISSUE TITLE
How to contribute? 

### ISSUE THREAD

**Issue by EmreSURK** [thomashiemstra/fred#3]
Great work! 
The arm has reverse kinematics and it is cool. 
Did you make all of these of your own? 

The project does not have a contribution guide, how can I contribute to the project?

**Comment by thomashiemstra** [comment]
Yes I made that function myself based on the book I linked. I'd suggest forking my project.
